lib/double_entry/reporting/aggregate.rb

```diff
@@ -93,16 +95,17 @@ def range_is_complete?
 
       def field_hash
         {
-          :function => function,
-          :account => account,
-          :code => code,
-          :year => range.year,
-          :month => range.month,
-          :week => range.week,
-          :day => range.day,
-          :hour => range.hour,
-          :filter => filter.inspect,
-          :range_type => range.range_type.to_s,
+          :function        => function,
+          :account         => account,
+          :code            => code,
+          :partner_account => partner_account,
```

Comment:
I've chatted to @pablolee and he's good with this now, the `field_hash` is used to identify cached aggregates and therefore we want to differentiate aggregates with no restriction on partner account (ie. `nil` partner_account) versus those with a filter on partner account.